app/src/main/java/fr/iut_rodez/pathpilot_android_client/login/TokenJWT.java

```diff
@@ -0,0 +1,66 @@
+package fr.iut_rodez.pathpilot_android_client.login;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import androidx.annotation.NonNull;
+
+import java.util.Date;
+
+public class TokenJWT implements Parcelable {
```

Comment:
Change to JWTToken to have english name and javadoc comments for the class

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/f495a2fcc43e0fc3d934ae46d71b9af6a0856994